### PR TITLE
Docs: fake discord server

### PR DESCRIPTION
**Pull request by hannibal002**
preview: https://github.com/hannibal002/SkyHanni/blob/fake-discord-server/docs/FAKE_DISCORD_SERVER.md

exclude_from_changelog

### CODE REVIEW

File: docs/FAKE_DISCORD_SERVER.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## I have got hacked / my account got stolen. What to do?
+
+### I cant log in into my account anymore
+
+You can try to get your account back by contacting microsoft support.
+You need to send them a ticket via your email, telling them what happened.
+Also provide them with exact information how and when you have bought your minecraft account.
+Good luck!
+
+### Someone else logged in into my account, but i can still access the account myself
```
Comment (CalMWolfs) [inline docs/FAKE_DISCORD_SERVER.md]:
```suggestion
### Someone else logged in into my account, but I can still access the account myself
```

File: docs/FAKE_DISCORD_SERVER.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Disable the multiplayer mode on xbox account settings to prevent others from logging in your account immediately.
+Try to log in on hypixel to kick the other user off from hypixel.
+Change your password on minecraft.net.
+Wait 24 hours and enable multiplayer mode again.
```
Comment (CalMWolfs) [inline docs/FAKE_DISCORD_SERVER.md:38]: I was told its longer than that to be safe, probably more like a week

File: docs/FAKE_DISCORD_SERVER.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+> The fake server has many channels without any sent message sent or without access to see the messages in the channel
+
+The real server has a lot of channels you can view and everyone of them has a lot of messages that (for some channels) even go back over a year.
+
+> The fake server does not link to anyhting outside itself.
+
+The real server has channels that link to github, to updates, to other ressources etc.
+
+> The fake server does not allow you to write in the chat.
```
Comment (CalMWolfs) [inline docs/FAKE_DISCORD_SERVER.md:54]: I think this is misleading as I do believe you can type in the channels (though some may be locked behind "verification") but the chat history is a good point

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
This should also link to https://hypixel.net/threads/all-about-skyblock-account-security.5231557/

**Review by Thunderblade73 — APPROVED** [review]
It does the job to inform about the fake server.

**Pull request closed without merging**